_pages/en/software-assessment-sharing-pilot.md

```diff
@@ -0,0 +1,136 @@
+
+## GCxchange Group
+
+GCxchange is an instance of Sharepoint/Teams making a good platform to create a shared document library and space to dicsuss software assessments in the GC.
+The GCexchange group is accessible to most of GC departments already.
+GC Employees can see what documents have been shared and participate in discussions about software assessments.
+
+Note: **Only Unclassified information can be shared on GCxchange** and that is what we want to share as part of the pilot.
+
+## Sharing ESDC Software Assessments
+
+Before software can be used at ESDC, it must be assessed for architecture, accessibility, security and privacy.
+The teams responsible for software assessments at ESDC agreed to share templates and the assessments they produce during the software intake process.
+There were legitimate concerns about what we can and can't share and if these are really protected or classified documents.
+
+### Document Classification
+
+Only the Security teams had most of their documents marked as Protected.
+Everything else was assumed Unclassified but when asked to share the documents there was some worry that they could be Protected.
+
+The Regional Security Office (RSO) reviewed some sample assessment documents from architecture, accessibility and security, and found that they were Unclassified.
+The RSO also reminded teams that classification of documents is still the responsibility of each team creating then.
+ 
+### Legal Questions
+
+As a starting position, the Government of Canada is one legal entity, hence sharing information between departments is generally not a problem.
+However, in order to not prejudice future procurement processes we must confirm the following before sharing:
+
+- The documents shared are for information purposes only - that is, to provide additional information for departments to consider when applying their own assessments for software usage;
+- the information is shared only inside the GC; 
+- the results of the shared audits do not disqualify a vendor from solicitation with other departments - that is, applying a blanket negative recommendation for all of GC would have important legal risks; and
+- if a particular vendor has a highly restrictive NDA, confirm with legal.
+
+Legal confirmed that there are no issues with ESDC sharing assessments, since this is for informational purposes only and it is not a blanket assessment for all GC departments to follow.
+This applies wether the documents are Unclassified, Protected or Classified.
+The diference would be the process to share these documents.
+
+For the pilot, we are only looking to share Unclassified information, since that is what is curently supported on GCxchange.
+
+### Fear of Being Embarrased
+
+More than one team claimed that sharing the documents they produce would not be useful to other departments.
```

Comment:
Should reword to use a more... friendly language :) Something like
"Through the pilot, some internal teams expressed concerns around usefulness of sharing information between departments, of fear of causing embarrassment, and questioning the legal and classification of those assessments"

The pilot was able to demonstrate that other departments, especially smaller ones that do not have the same capacity as ESDC, were pleased to have ESDC's assessments shared; that no embarrassment occurred, and that only unclassified documents were shared with the legal cover from ESDC's Legal Services (is that their name?)"